tests/scripts/validate_modified_files.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env -S bash
-set -ex
```

Comment:
any reason for removing this? Could be useful for debugging and to exit immediately if shell returns non zero.

Reply:
@sp98  wrote:

> any reason for removing this? Could be useful for debugging and to exit immediately if shell returns non zero.

Here are the reasons:


I prefer to let scripts exit explicitly on error conditions istead of implicitly because of `set -e`

explicit exit makes it easier to log exactly why the script exited. Otherwise it is obfuscated.

Additionally, For me the `set -x`tracing makes it extremely difficult to see the script's intentional output. So for me this makes debugging much more difficult instead of helping. 


This change is also part of PR #14672  and not originally of this one.